### PR TITLE
Test PR

### PR DESCRIPTION
**Pull request by adarshdubey-star**
# Description

FIXES: 

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
Very :green_apple: :)

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 ## Getting Started
 
-The **edge-api** project is an API server for fleet edge management capabilities. The API server will provide [Restful web services](https://www.redhat.com/en/topics/api/what-is-a-rest-api).
+The **edge-api** project is an API server for fleet edge management capabilities. The API server will provide [Restful web services](https://www.redhat.com/en/topics/api/what-is-a-rest-api)
```
Comment (loadtheaccumulator) [inline README.md:13]: It's been around three years. We should consider changing it to "The API server provides..." :-p

**Pull request closed without merging**